hack/e2e_compare.sh

```diff
@@ -0,0 +1,14 @@
+#!/usr/bin/env bash
+
+set -e
+
+# Compare 2 files generated by e2e_collect. This ensures elements are sorted by version
+
```

Comment:
Please add a check to ensure the argument(s) are passed in.

Reply:
Fixed

Reply:
Where?

Reply:
Sorry, forgot to push it. `test "$#" -ne 2 && echo "Usage: ./e2e_compare.sh FILE1 FILE2" && exit 1` would implement that check